conf/distro/acrn-demo-sos.conf

```diff
@@ -11,3 +11,12 @@ LINUX_ACRN_APPEND ?= "hvlog=2M@0x1FE00000"
 LINUX_GVT_APPEND ?= "i915.enable_gvt=1 i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.domain_scaler_owner=0x011100"
 
 APPEND += "${LINUX_ACRN_APPEND} ${LINUX_GVT_APPEND}"
+
+GRUB_BUILDIN_append = " chain "
```

Comment:
Direct boot mode use GRUB, in that case, how this chain-loading will work. Could you please check that ?
https://projectacrn.github.io/latest/introduction/index.html#direct-boot-mode

Reply:
current state only support for De-privilege boot mode.
Direct boot mode required multiboot module in grub-efi.
in that case  chain module are not required but multiboot module and grub menu with two entries one to direct boot, another one boot to native